rerun_py/rerun_sdk/rerun/components/media_type_ext.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Final
+
+if TYPE_CHECKING:
+    from . import MediaType
+
+
+# TODO(#2388): constants in fbs
+class MediaTypeExt:
+    TEXT: Final = "text/plain"
+    MARKDOWN: Final = "text/markdown"
+    GLB: Final = "model/gltf-binary"
+    GLTF: Final = "model/gltf+json"
+    OBJ: Final = "model/obj"
+
+    # TODO(cmc): these should just be constants, but circular import hell...
+
+    @staticmethod
+    def plain_text() -> MediaType:
+        """Plain text: `text/plain`."""
+        from . import MediaType
+
+        return MediaType(MediaTypeExt.TEXT)
+
+    @staticmethod
+    def markdown() -> MediaType:
```

Comment:
What is more pythonic - users typing `MediaType.markdown()` or `MediaType.MARKDOWN`? 🤔 

I don't think we need to support both just because we do so in e.g. Rust (where we need to for technical reasons)

Reply:
They're not the same though. I would love to get rid of these methods, but it's the only way I can think of to actually provide a properly type constant without ending up in circular import hell.